source/appModuleHandler.py

```diff
@@ -487,6 +487,18 @@ def isBadUIAWindow(self,hwnd):
 		"""
 		return False
 
+	def shouldProcessUIAPropertyChangedEvent(self, sender, propertyId):
+		"""
+		Determines whether NVDA should process a UIA property changed event.
+		Returning False will cause the event to be dropped completely. This can be
+		used to work around UIA implementations which flood events and cause poor
+		performance.
+		Returning True means that the event will be processed, but it might still

```

Comment:
can you briefly describe what these params mean so I can send a followup to document them?